src/Controller/CRUDController.php

```diff
@@ -1270,7 +1270,7 @@ protected function redirectTo($object)
                     $url = $this->admin->generateObjectUrl(
                         $route,
                         $object,
-                        $request->request->has('_tab') ? ['_tab' => $request->request->get('_tab')] : []
+                        '' !== $request->request->get('_tab', '') ? ['_tab' => $request->request->get('_tab')] : []
```

Comment:
There is a shorter solution:
```suggestion
                        array_filter(['_tab' => $request->request->get('_tab')])
```